glyphs-reader/plist_derive/src/attrs.rs

```diff
@@ -1,26 +1,60 @@
-use syn::{spanned::Spanned, Attribute};
+use syn::{
+    punctuated::Punctuated, spanned::Spanned, Attribute, Expr, ExprLit, Lit, Meta, MetaNameValue,
+    Token,
+};
 
-/// Attributes on fields
-#[derive(Clone, Default)]
-pub(crate) struct FieldAttrs {
-    pub(crate) rest: Option<syn::Path>,
-    pub(crate) default: Option<syn::Path>,
-}
+static IGNORE: &str = "ignore";
+static KEY: &str = "key";
+static ALT_NAME: &str = "alt_name";
+static DOC: &str = "doc";
 
-static REST: &str = "rest";
-static DEFAULT: &str = "default";
+#[derive(Clone, Default, Debug)]
+pub(crate) struct FromPlistFieldAttrs {
```

Comment:
if we now only have the one item let's just keep the old name?

Reply:
done